include/pm/policy.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_PM_POLICY_H_
+#define ZEPHYR_INCLUDE_PM_POLICY_H_
+
+#include <pm/pm.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Function to get the next PM state based on the ticks
```

Comment:
nit: missing `@param ticks` and `@return` docs

Reply:
Yep, I just moved it around. Let me add a new commit fixing it.